core/src/components/datetime/test/multiple/datetime.e2e.ts

```diff
@@ -158,10 +158,16 @@ configs({ modes: ['md'], directions: ['ltr'] }).forEach(({ title, config }) => {
       });
     });
 
-    test('multiple default values across months should display at least one value', async () => {
-      const datetime = await datetimeFixture.goto(config, MULTIPLE_DATES_SEPARATE_MONTHS);
+    test('should not scroll to new month when value is updated with dates in different months', async ({ page }) => {
```

Comment:
I didn't add a test for ensuring it *does* scroll when all dates are in the same month because that's already covered by [this screenshot test](https://github.com/ionic-team/ionic-framework/blob/09bc77eb3dcca2b28f35b714a1cef64340917365/core/src/components/datetime/test/multiple/datetime.e2e.ts#L86).

Reply:
Ah, true; I'd assumed loading with the values already set was impossible in pure JS since you have to do it in a separate `<script>` tag, but looking at a React app with the `value` prop initialized, the behaviors are equivalent. I'll add a new test 👀

Reply:
Added in https://github.com/ionic-team/ionic-framework/pull/28847/commits/4369f2adda981eb15f2c9a86f9d6e818812a10fd.